radicle/src/profile.rs

```diff
@@ -32,6 +34,14 @@ pub mod env {
     pub const RAD_SOCKET: &str = "RAD_SOCKET";
     /// Passphrase for the encrypted radicle secret key.
     pub const RAD_PASSPHRASE: &str = "RAD_PASSPHRASE";
+
+    pub fn read_passphrase() -> Option<super::Passphrase> {
+        let Ok(passphrase) = std::env::var(RAD_PASSPHRASE) else {
+            return None;
+        };
+
+        Some(super::Passphrase::from(passphrase.trim_end().to_owned()))
```

Comment:
@cloudhead should we remove the 'trim_end' here?  Some people may have passphrases ending in spaces.

Reply:
:+1: 

It's coming from an env var here, but yeah we can remove the trim.